tests/console/snapper_jeos_cli.pm

```diff
@@ -47,6 +45,15 @@ sub rollback_and_reboot {
     }
     select_console('root-console');
     assert_script_run("snapper list");
+    # check whether SUSEConnect --rollback is running executed by rollback-reset-registration
+    # this might cause a system management lock by zypper
+    for (my $runs = 0; $runs < 5; $runs++) {
+        if (script_run('test -f /var/lib/rollback/check-registration') == 1) {
```

Comment:
Will this be enough? Shouldn't we also check for zypper process?
In your VR, I see zypper running  in the ps output:
http://kepler.suse.cz/tests/14494#step/snapper_jeos_cli/104

Reply:
Ah ok, got it. If that file doesn't exist, the rollback is supposed to be completed and no more actions will be performed.

Reply:
https://github.com/openSUSE/rollback-helper/blob/master/sbin/rollback-reset-registration#L30 

I think it should be, as the file only exists while `SUSEConnect --rollback` hasn't successfully finished.